test/suites/devlxd.sh

```diff
@@ -1,4 +1,8 @@
 test_devlxd() {
+  if [ "$(lxc image info testimage | awk '/^Cached:/ {print $2}')" = "yes" ]; then
+    echo "==> Deleting cached testimage to ensure testimage will not be accessible through devlxd"
+    lxc image delete testimage
+  fi
   ensure_import_testimage
```

Comment:
wont this just reimport it here again?

Reply:
if there is a cached image available could we test that /dev/lxd can access it before we remove it?

Reply:
The cached image is one that lingers from a previous test.
The one imported by `ensure_import_testimage` is not cached.